libraries/AP_Scripting/applets/RockBlock.lua

```diff
@@ -54,12 +54,12 @@ function bind_param(name)
  end
 
  -- setup RCK specific parameters
-assert(param:add_table(PARAM_TABLE_KEY, PARAM_TABLE_PREFIX, 4), 'could not add param table')
+assert(param:add_table(PARAM_TABLE_KEY, PARAM_TABLE_PREFIX, 5), 'could not add param table')
```

Comment:
Isn't this going to break people's working setups?

Reply:
Tested this on a CubeOrangePlus - the params weren't lost when upgrading.